src/Worker/Core/Shims/TaskOrchestrationContextWrapper.cs

```diff
@@ -177,10 +177,14 @@ public override async Task<TResult> CallSubOrchestratorAsync<TResult>(
         static string? GetInstanceId(TaskOptions? options)
             => options is SubOrchestrationOptions derived ? derived.InstanceId : null;
         string instanceId = GetInstanceId(options) ?? this.NewGuid().ToString("N");
+        string defaultVersion = this.invocationContext.Options != null && this.invocationContext.Options.Versioning != null
```

Comment:
Null operators may help here:

```suggestion
        string defaultVersion = this.invocationContext.Options?.Versioning?.DefaultVersion ?? string.Empty;
```